lib/WebpackOptionsDefaulter.js

```diff
@@ -166,25 +170,16 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 		this.set("node.__filename", "mock");
 		this.set("node.__dirname", "mock");
 
-		this.set(
-			"performance",
-			"make",
-			options => (isProductionLikeMode(options) ? false : undefined)
```

Comment:
The original logic seem to be broken anyway (reversed).

Reply:
Yeah It has no effect here, overridden by the `call` below.